src/Aserto.Clients/Options/AsertoDirectoryOptions.cs

```diff
@@ -40,7 +40,19 @@ public AsertoDirectoryOptions()
         /// <param name="insecure">Bool indicating whether insecure service connections are allowed when using SSL.</param>
         public AsertoDirectoryOptions(string serviceURL = "", string readerURL = "", string writerURL = "", string importerURL = "", string exporterURL = "", string modelURL = "", string apiKey = "", string tenantID = "", bool insecure = false)
```

Comment:
Should there be support for `NoTLS` aka `plaintext`, besides `insecure`?

Reply:
Prepared this in another PR on top of these changes https://github.com/aserto-dev/aserto-dotnet/pull/43